### ISSUE TITLE
Possible Solution for Encryption on Exit

### ISSUE THREAD

**Issue by tdadvocate** [mmiksaa/obsidian-protected-note#6]
Apologies in advance if this has already been thought of and it did not work. I don't have enough experience with Obsidian yet and only just started using it on Android today. But I was wondering if perhaps there is a way to keep the encryption active by switching to another vault via the ribbon button. It wouldn't be a perfect solution, but it could allow for vaults to stay encrypted apart from one another perhaps. Sorry again if this was already attempted and thank you for the plugin regardless!

**Comment by mmiksaa** [comment]
Sorry for the late response. I didn't actually get what you mean. Could you explain it more openly?

**Comment by fhera** [comment]
Hi!

I think @tdadvocate found a bug.

I mean, if I close obsidian without clicking the lock in the left bar, the notes will not be encrypted and I can read them until I open obsidian again.

**Comment by mmiksaa** [comment]
Yes you're right. I did it because plugin is supported on phones. Obsidian doesn't let start a function after you close the app on phones (on pc it works well). So encryption should start after you "submit" the encryption.

Also i'm thinking about to make available encryption only for pc. As i noticed most of users who needs encryption use pc only. If i make it pc only my plugin will be way better.
